Sources/PenguinTesting/StdlibProtocolTests.swift

```diff
@@ -255,20 +250,16 @@ extension RandomAccessCollectionChecker: SequenceRefinementChecker
 }
 
 extension Sequence where Element: Equatable {
-  ///  XCTests that `self` has proper semantics for the known refinements of `Sequence` to which
-  ///  `Self` has been declared to conform.
+  /// XCTests that `self` has proper semantics for the known refinements of `Sequence` to which
+  /// `Self` has been declared to conform.
```

Comment:
This is a lie now that you removed the mutable collection functionality.

Reply:
Depending on what "known refinements" means it's not a lie :) But doesn't matter if we revert it.